frontend/tests/integration/components/transportation/study-area-map/census-tract-selector-test.js

```diff
@@ -0,0 +1,55 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, settled } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+
+module('Integration | Component | transportation/study-area-map/census-tract-selector', function(hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+
+  test('it adds a selected feature geoId to the analysis model study selection', async function(assert) {
```

Comment:
Judging from the test names and the `if when then` phrases, this component shoudl be renamed because it's not really selecting anything, no?

Reply:
see above :)